index.html

```diff
@@ -1,167 +1,278 @@
+              <li><img src="/img/social icons/Linkedin_icon.svg" alt="" /></li>
+              <li><img src="/img/social icons/medium_icon.svg" alt="" /></li>
+            </ul>
+            <a href="#" class="connect-two">LET'S CONNECT</a>
+          </div>
+          <br />
+          <button class="btn-primary">Get my resume</button>
+          <div class="modal">
+            <ul class="modal-items">
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Languages
+                  <img src="/img/Icon - Chevron - Down - Larger.png" alt="" />
+                </h3>
+                <br />
+                <span class="item">
+                  <img src="/img/js.png" alt="" />
+                  <h4>Javascript</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/html5.png" alt="" />
+                  <h4>HTML</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/css3.png" alt="" />
+                  <h4>CSS</h4></span
+                >
+                <br />
+              </li>
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Frameworks <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Skills <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact">
+        <div class="contact-container">
+          <div class="contact-desc">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <div class="form">
+            <form
+              id="fs-frm"
+              name="simple-contact-form"
+              accept-charset="utf-8"
+              action="https://formspree.io/f/xnqrdrre"
+              method="post"
+            >
+              <fieldset id="fs-frm-inputs">
+                <input
+                  type="text"
+                  name="name"
+                  id="full-name"
+                  required=""
+                  placeholder="Raj"
+                  maxlength="30"
+                />
+
+                <input
+                  type="email"
+                  name="_replyto"
+                  id="email-address"
+                  required=""
+                  placeholder="rajbalaji003@gmail.com"
+                  maxlength="30"
+                />
+
+                <textarea>
+                  rows="5"
+                  name="message"
+                  id="message"
+                  placeholder="Write your message"
```

Comment:
I suggest you put this as attributes of the `<textarea>`, that is they should come before the closing tag `>`.
instead try: 
...`< textarea rows="5"
                  name="message"
                  id="message"
                  placeholder="Write your message" 
                  ...
                  >`

 It has caused your code to be buggy.
This is its result:
<img width="367" alt="Screenshot 2022-09-28 at 12 03 39" src="https://user-images.githubusercontent.com/48546992/192737841-d65f6e99-bd47-4851-84cc-8e9e27cdb10e.png">